### PR TITLE
Fix non-editable order message

### PR DESCRIPTION
**Pull request by rcdtroly**
### Why:
* We currently show an error message and use 'the order your attempted to modify can no longer be edited.' 
### What:
* update error message base on status
### Refs:

https://app.asana.com/0/939825968139797/1113808830327509

### CODE REVIEW

**Comment by JamesEmpireOne** [comment]
Does this apply to only admins @rcdtroly or does this also apply to what customers see on the frontend?

**Comment by rcdtroly** [comment]
> Does this apply to only admins @rcdtroly or does this also apply to what customers see on the frontend?

@JamesEmpireOne  only for admin users

File: includes/frontend/controllers/class-wp99234-orders.php [hunk 1]
```diff
@@ -246,5 +245,32 @@ public function wp99234_edit_order_check($order_action = 'edit-order', $troly_or
...
+  {
+    switch ($status) {
+        case "paid":
+            $message = "Order is in progress and cannot be edited";
+            break;
+        case "completed":
+            $message = "Order is fulfilled and can no longer be edited";
+            break;
+        case "cancel":
+            $message = "Order is canceled and cannot be edited";
```
Comment (atifempireone) [inline includes/frontend/controllers/class-wp99234-orders.php]: typo - cancelled

File: includes/frontend/controllers/class-wp99234-orders.php [hunk 2]
```diff
@@ -246,5 +245,32 @@ public function wp99234_edit_order_check($order_action = 'edit-order', $troly_or
...
+
+  /**
+     * Message for non-editable orders
+     * @since 2.9
+     * @package Troly
+     */
+  private static function wp99234_order_status_error_notice($status)
+  {
+    switch ($status) {
+        case "paid":
```
Comment (atifempireone) [inline includes/frontend/controllers/class-wp99234-orders.php]: All messages below should start with:

"The order you attempted to modify is.."

**Comment by rcdtroly** [comment]
@atifempireone fixed typo

**Pull request merged**